recipe/meta.yaml

```diff
@@ -121,6 +121,20 @@ outputs:
   - name: clangdev
     script: install_clangdev.sh  # [unix]
     script: install_clangdev.bat  # [win]
+    files:
+      include:
+        - include/clang-c                    # [unix]
```

Comment:
Unfortunately, even the newer `files.include` will still unconditionally clobber files (rather than using the much more sensible default of not clobbering; which could still be overridden with `always_include_files`); there's no way to use this feature in a way that doesn't clobber by default, which is a massive missed [opportunity](https://github.com/conda/conda-build/issues/5455)

Reply:
Though in this case, that particular detail is less relevant, since we've removed all the host deps that would otherwise participate in the snapshotting mechanism (that `files` & `files.include` ignores)

Reply:
Yeah, `files` vs `files.include` doesn't matter. I'm okay with using `files` for `clangdev` so that we don't accidentally package something that belongs in a different package. (I noticed that some files that should be in `clang-tools` was included in clangdev in windows that I fixed) This doesn't fix the issue when files get missed, but that's something we'll have to live with I guess.